src/Services/GTT/GTT.Api/Configuration/Routes.cs

```diff
@@ -29,6 +29,14 @@ public static class Routes
     /// </summary>
     #region ExerciseManagement
     public const string ExerciseLibrary = "v1/excercise-library";
+    public const string ExerciseLibraryImage = "v1/excercise-library/upload-image";
```

Comment:
Why do we need both, ExerciseLibraryImage and UploadImage(line 40).
I think the route name UploadImage is more descriptive